src/scroll-timeline-css.js

```diff
@@ -140,8 +140,8 @@ function updateKeyframesIfNecessary(anim, options) {
 
 export function initCSSPolyfill() {
   // Don't load if browser claims support
-  if (CSS.supports("animation-timeline: works")) {
-    return;
+  if (CSS.supports("animation-timeline: none")) {
```

Comment:
"animation-timeline: --works" is probably a better choice. Strictly speaking a null timeline can be supported even without scroll-driven animation support.